vms/proposervm/vm.go

```diff
@@ -727,3 +768,9 @@ func (vm *VM) cacheInnerBlock(outerBlkID ids.ID, innerBlk snowman.Block) {
 		vm.innerBlkCache.Put(outerBlkID, innerBlk)
 	}
 }
+
+// writeAcceptedSlotMetrics use the previously stored slot index and add that to the
+// metrics.
+func (vm *VM) writeAcceptedSlotMetrics(slot uint64) {
+	vm.acceptedBlocksSlotHistogram.Observe(float64(slot))
+}
```

Comment:
Since this is only called in one place (and it's a one-liner), it feels like this could just be in-lined.